src/Data/Characters/Razor/index.tsx

```diff
@@ -298,13 +295,13 @@ const sheet: ICharacterSheet = {
       text: trm("fullBurstDMG.description"),
     }, ct.headerTemplate("burst", {
       fields: [{
-        node: infoMut(dmgFormulas.burst.fullBurstDmg1, { key: `char_${key}:fullBurstDMG.dmg1` })
+        node: infoMut(dmgFormulas.burst.fullBurstDmg1, { key: `char_${key}:fullBurstDMG.dmg1`, variant: "invalid" })
```

Comment:
is there any point in keeping these since we have multi-target?? lets keep the "simple" targets as truly simple targets, or we have too much nesting.